.github/CONTRIBUTING.md

```diff
@@ -71,6 +70,18 @@ Explain local setup
 Explain CI setup
 ```
 
+In most cases the `<type>` should be one of the following:
```

Comment:
As said on Slack, I agree with Gary. I personally would not be able to adhere to the `<type>` requirement 80% of the time. The simple fact of always including a unit test with a change makes this impossible.